src/utils/arrayHasElements.ts

```diff
@@ -0,0 +1,3 @@
+export const arrayHasElements = <T>(arr: T[] | undefined): arr is T[] => {
+  return !!arr && arr.length > 0;
+};
```

Comment:
you should reuse the lodash `isEmpty` as we already use it

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1977/commits/fdf67dbe5bdeeaf6d08d0c682dace7fdb4e3a761